lace/scanalign_option.py

```diff
@@ -0,0 +1,131 @@
+import abc
```

Comment:
This file does not belong in lace. It has nothing to do with mesh handling. Please put it in tfsmpl.